circus-train-housekeeping/src/main/java/com/hotels/bdp/circustrain/housekeeping/repository/CircusTrainLegacyReplicaPathRepository.java

```diff
@@ -18,11 +18,10 @@
 import org.springframework.context.annotation.Lazy;
 import org.springframework.transaction.annotation.Transactional;
 
-import com.hotels.bdp.circustrain.housekeeping.model.CircusTrainLegacyReplicaPath;
+import com.hotels.housekeeping.model.HousekeepingLegacyReplicaPath;
```

Comment:
I think we could get rid of this interface and move the annotations to the base interface, let me know what you think

Reply:
Sounds like a good idea to me, simpler for end users and the base interface is already couled to spring data package via imports.